actions/setup-nix-gati/action.yml

```diff
@@ -0,0 +1,77 @@
+name: setup-nix-gati
+description: "Setup the environment for Nix using GATI"
+
+inputs:
+  # general inputs
+  aws-role-arn:
+    description: ARN of role capable of getting token from GATI
+    required: true
+  aws-lambda-url:
+    description: URL of GATI lambda function
+    required: true
+  aws-region:
+    description: AWS region
+    required: true
+  aws-role-duration-seconds:
+    description: Duration of role in seconds
+    required: false
+    default: "900"
+  enable-magic-cache:
+    description: "Enable magic nix cache"
+    required: false
+    default: false
+  # grafana inputs
+  metrics-job-name:
+    description: "grafana metrics job name"
+    required: false
+    default: setup-nix-gati
+  gc-host:
+    description: "grafana hostname"
+    required: false
+  gc-basic-auth:
+    description: "grafana basic auth"
+    required: false
+  gc-org-id:
+    description: "grafana org/tenant id"
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - name: Setup GitHub token
+      id: setup-github-token
+      uses: smartcontractkit/.github/actions/setup-github-token@main
```

Comment:
This should either be:
1. tagged to a commit (not `@main`) or
2. referenced as local action (after checking out this repo)

Reply:
good catch!